it/v1/endpoints/AmendPeriodSummaryControllerISpec.scala

```diff
@@ -30,79 +30,41 @@ import v1.stubs.{AuthStub, DownstreamStub, MtdIdLookupStub}
 
 class AmendPeriodSummaryControllerISpec extends IntegrationBaseSpec with JsonErrorValidators with AmendPeriodSummaryFixture {
 
-  val requestBodyJson: JsValue           = amendPeriodSummaryBodyMtdJson
```

Comment:
What formatting do you have setup, as the Scalafmt config normally would keep the equals signs aligned.